ligo-webide-frontend/ligo-ide/src/scss/index.scss

```diff
@@ -46,8 +46,9 @@ $jumbotron-padding: 1.5rem;
 $modal-md: 720px;
 $modal-content-bg: $background;
 $modal-header-border-color: $black;
-$modal-backdrop-opacity: .7;
-$btn-disabled-opacity: .3;
+$modal-backdrop-opacity: .6;
+$modal-backdrop-bg: #d0e3ff;
```

Comment:
where did you get this hexcode (and the other hexcode literals)?

Reply:
Used ligo blue colour but more soft, so it was just random softer color based on ligo blue color